executor/aggfuncs/func_avg.go

```diff
@@ -136,6 +178,62 @@ func (e *avgPartial4Decimal) UpdatePartialResult(sctx sessionctx.Context, rowsIn
 	return nil
 }
 
+func (e *avgPartial4Decimal) Slide(sctx sessionctx.Context, rows []chunk.Row, lastStart, lastEnd uint64, shiftStart, shiftEnd uint64, pr PartialResult) error {
```

Comment:
Window function will only be used as `aggregation.CompleteMode`, so that this will never be hit.

Reply:
I will remove them.